test/repl.jl

```diff
@@ -72,6 +72,9 @@ temp_pkg_dir() do project_path; cd(project_path) do; mktempdir() do tmp_pkg_path
     tokens = Pkg.REPLMode.tokenize("add git@github.com:JuliaLang/Example.jl.git@0.5.0")
     @test tokens[1][2] == "git@github.com:JuliaLang/Example.jl.git"
     @test repr(tokens[1][3]) == "VersionRange(\"0.5.0\")"
+    tokens = Pkg.REPLMode.tokenize("add git@gitlab.fsl.jsc.näsan.guvv.drats/URGA2010.jl.git@0.5.0")
```

Comment:
Is this using some http protocol?
or there's typo for `git@gitlab.fsl.jsc.näsan.guvv:drats/URGA2010.jl.git`

Reply:
Yes, I wrote it wrong